### PR TITLE
tracing: fixed panic because of nil sampler

### PR DESCRIPTION
**Pull request by xBazilio**
Signed-off-by: Vasiliy Rumyantsev <4119114+xBazilio@users.noreply.github.com>

* [x] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

### CODE REVIEW

File: pkg/tracing/jaeger/config_yaml.go [hunk 1]
```diff
@@ -141,6 +141,7 @@ func getSampler(config Config) tracesdk.Sampler {
 		sampler = jaegerremote.New(config.ServiceName, remoteOptions...)
 	// Fallback always to default (rate limiting).
 	case SamplerTypeRateLimiting:
+		fallthrough
```
Comment (fpetkovski) [inline pkg/tracing/jaeger/config_yaml.go:144]: Should we simply remove the entire `SamplerTypeRateLimiting` case, or do you want to keep it here for documentation reasons?

Comment (xBazilio) [reply pkg/tracing/jaeger/config_yaml.go:144]: I think it would be nice to keep it, yes.

**Review by fpetkovski — DISMISSED** [review]
LGTM.

cc @douglascamata

**Review by douglascamata — CHANGES_REQUESTED** [review]
LGTM, but missing the changelog entry. Could you please add it, @xBazilio?

**Review by douglascamata — DISMISSED** [review]
LGTM

**Comment by xBazilio** [comment]
I had to repush to sign the comit

**Comment by xBazilio** [comment]
go / Thanos end-to-end tests (4, 2) (pull_request) 
for some reason failing, though I didn't do anything with it.

**Comment by douglascamata** [comment]
@xBazilio no worries. There are some flaky tests. The maintainers and many contributors are aware. 👍

**Review by fpetkovski — APPROVED** [review]

**Review by douglascamata — APPROVED** [review]

**Review by yeya24 — APPROVED** [review]
Thanks for fixing it!

**Pull request merged**